datalad/cmdline/support.py

```diff
@@ -0,0 +1,85 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""
+"""
+
+__docformat__ = "restructuredtext"
+
+from datalad.utils import swallow_outputs
+from datalad.api import wtf
+import sys
+import os
+
+from logging import getLogger
+
+lgr = getLogger("datalad.cmdline")
+
+os.environ["MESSAGELEVEL"] = "QUIET"
+os.putenv("MESSAGELEVEL", "QUIET")
+
```

Comment:
Why is this process wide upon import change of this environment variables is needed? Ideally should be avoided, or if really needed - a comment why it is needed is needed

Reply:
helpme has a message level controlled via environment, and that's how you disable warnings. if you don't put that there will be an ugly WARNING no GitHub token.... every time the helpme_submit function is used.

Reply:
I'll add a note.

Reply:
Then it should be patched only around helpme invocation . I can do that later. I would also recommend in helpme to prepend it with HELPME_ since it sounds to be specific to it

Reply:
Yeah that was bothering me too :) I can do a quick release for that and I'll update the PR here.

Reply:
okay pushing update shortly.